checkoutmanager/runner.py

```diff
@@ -22,6 +22,8 @@
     'co': "Grab missing checkouts from the server",
     'missing': "Print directories that are missing from the config file",
     'out': "Show changesets you haven't pushed to the server yet",
+    'cu': "Check for updates that will be pulled in with 'up'",
```

Comment:
"cu" was supposed to be "check for updates". I'm fine with "in", and it makes sense to use "in" instead since there's also an "out". I'll make the change.